files/etc/nginx/sites-available/default

```diff
@@ -0,0 +1,54 @@
+upstream rails_app {
+  server localhost:3000 fail_timeout=0;
+}
+
+server {
+  listen 80 default_server;
+  listen [::]:80 default_server;
+  listen 443 ssl default_server;
+  listen [::]:443 ssl default_server;
+  server_name _;
+
+  ssl_certificate /etc/ssl/private/workarea.crt;
+  ssl_certificate_key /etc/ssl/private/workarea.pem;
+
+  # define the public application root
+  root   /srv/shop/public;
+  index  index.html;
+  # define where Nginx should write its logs
+  access_log stdout;
+  error_log stderr;
+
+  # deny requests for files that should never be accessed
+  location ~ /\. {
+     deny all;
+  }
+  location ~* ^.+\.(rb|log)$ {
+     deny all;
+  }
+
+  # serve static (compiled) assets directly if they exist (for rails production)
+  location ~ ^/(assets|images|javascripts|stylesheets|swfs|system)/   {
+     try_files $uri @rails;
+     access_log off;
+     gzip_static on;       # to serve pre-gzipped version
+     expires max;
+     add_header Cache-Control public;
+
+     add_header Last-Modified "";
+     add_header ETag "";
```

Comment:
Does this override and blank out these headers? That seems bad?

Reply:
it would seem so. sorry i missed this, was mostly a copy-paste from something i did prior.